robottelo/manifests.py

```diff
@@ -221,8 +221,6 @@ def upload_manifest_locked(org_id, manifest=None, interface='API', timeout=None)
         subscription_id = result[id']
     """
 
-    if interface not in ['API', 'CLI']:
```

Comment:
Instead we should remove this method itself right ?

Reply:
robottelo.manifests is gone but It would be good to keep both interfaces.